src/scripts/modules/navigation/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import $gfwdom from '../../facade';
+import utils from '../../utils';
+import jstpl from 'micro-template';
+import navigationTpl from './navigation.tpl';
+
+import globaljson from './global.json';
+import climatejson from './climate.json';
+import howtojson from './howto.json';
+
+const navigationMenus = {
+  'default': globaljson,
+  'localhost': globaljson,
+  'climate.globalforestwatch.org': climatejson,
+  'howto.globalforestwatch.org': howtojson
+};
+
+
+/**
+ * Navigation
+ * @param  {window} root
+ * @return {Class}
+ */
+class Navigation {
+
+  constructor() {
+    this.$el = $gfwdom('#submenuMenuMobile');
+    if (!this.$el) {
+      throw new Error('element #submenuMenuMobile doesn\'t exist');
+    }
+    this.render();
+  }
+
+  render() {
+    var menu = navigationMenus[location.hostname] || navigationMenus['default'];
```

Comment:
Beautiful! But, one comment more: use navigationMenus.default instead of navigationMenus['default'].